### PR TITLE
[bitnami/grafana operator] Make the securityContexts more customizable.

### PR DESCRIPTION
**Pull request by Cellebyte**
**Description of the change**

**Benefits**

Make `containerSecurityContext` and `podSecurityContext` more user friendly.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/grafana-operator/values.yaml [hunk 1]
```diff
@@ -102,6 +102,9 @@ operator:
     enabled: true
     runAsUser: 1001
     runAsGroup: 1001
+    capabilities:
+      drop:
+      - ALL
```
Comment (pablogalegoc) [inline bitnami/grafana-operator/values.yaml:107]: Why are we dropping all capabilities when the containers are running by default as non-root and also setting the `allowPrivilegeEscalation` to false?

**Review by pablogalegoc — COMMENTED** [review]
Hi @Cellebyte, thanks for your contribution! Check out the comment I left when you have time

File: bitnami/grafana-operator/values.yaml (hunk 1 above)
Comment (Cellebyte) [reply bitnami/grafana-operator/values.yaml:107]: I just added it to comply with our policies which ask for that. It is not needed and if I should not add it we can drop it from the `securityContext`

Comment (pablogalegoc) [reply bitnami/grafana-operator/values.yaml:107]: Then in this case I would remove it just to keep the PR only related to the configurability improvements. Thank you very much for your work!

Comment (Cellebyte) [reply bitnami/grafana-operator/values.yaml]: Should i bump then only the patch release?
This is not changing the behaviour of the `values.yaml` interface

Comment (pablogalegoc) [reply bitnami/grafana-operator/values.yaml]: Yes, just bump the patch version. Thank you!

**Pull request merged**